### PR TITLE
Modify The Consul cluster nodes 's Description 

### PR DESCRIPTION
**Pull request by seifeHu**
'Consul cluster node ' can be replaced by 'The consul cluster agents'

Please answer these questions before submitting pull request

- Why submit this pull request?
- [ ] Bug fix
- [ ] New feature provided
- [ ] Improve performance

- Related issues

___
### Bug fix
- Bug description.

- How to fix?

___
### New feature or improvement
- Describe the details and related test reports.

### CODE REVIEW

**Comment by wu-sheng** [comment]
Related to #4021

File: docs/en/setup/backend/backend-cluster.md [hunk 1]
```diff
@@ -85,7 +85,7 @@ the yml to active.
 cluster:
   consul:
     serviceName: ${SW_SERVICE_NAME:"SkyWalking_OAP_Cluster"}
-    # Consul cluster nodes, example: 10.0.0.1:8500,10.0.0.2:8500,10.0.0.3:8500
+    # The consul cluster agents(agent is server agent or client agent). example: 127.0.0.1:8500,10.0.0.2:8500,10.0.0.3:8500
```
Comment (wu-sheng) [inline docs/en/setup/backend/backend-cluster.md]: If you are going to config the agent `ip:port`, then is it still necessary to have multiple addresses?  Or those addresses should be server agent addresses?

Comment (wu-sheng) [reply docs/en/setup/backend/backend-cluster.md]: `127.0.0.1:8500` and `10.0.0.2:8500` seems strange together. Should change this to 
```
example, 1. client agent, 127.0.0.1:8500, 2. server agent, 127.0.0.1:8500,10.0.0.2:8500,10.0.0.3:8500
```

Comment (seifeHu) [reply docs/en/setup/backend/backend-cluster.md]: ok. I will modify it

**Comment by wu-sheng** [comment]
Replaced by #4030 4030

**Pull request closed without merging**